gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/apim/core/api/model/mapper/ApiMigration.java

```diff
@@ -111,29 +111,45 @@ private MigrationResult<io.gravitee.definition.model.v4.Api> apiDefinitionHttpV4
         // TODO handle flows
         FlowExecution flowExecution = null;
```

Comment:
It is not related to this PR, but we should remove that TODO and inline the flowExcecution variable as we are dealing with the migration L135